Stronger/Onboarding/Welcome.swift

```diff
@@ -21,27 +21,35 @@ struct Welcome: View {
             areas: [
                 OnboardingInformationView.Content(
                     icon: {
-                        Image(systemName: "apps.iphone")
+                        Image(systemName: "fork.knife")
                             .accessibilityHidden(true)
                     },
                     title: "WELCOME_AREA1_TITLE",
                     description: "WELCOME_AREA1_DESCRIPTION"
                 ),
                 OnboardingInformationView.Content(
                     icon: {
-                        Image(systemName: "shippingbox.fill")
+                        Image(systemName: "figure.strengthtraining.functional")
                             .accessibilityHidden(true)
                     },
                     title: "WELCOME_AREA2_TITLE",
                     description: "WELCOME_AREA2_DESCRIPTION"
                 ),
                 OnboardingInformationView.Content(
                     icon: {
-                        Image(systemName: "list.bullet.clipboard.fill")
+                        Image(systemName: "figure.run")
                             .accessibilityHidden(true)
                     },
                     title: "WELCOME_AREA3_TITLE",
                     description: "WELCOME_AREA3_DESCRIPTION"
+                ),
+                OnboardingInformationView.Content(
```

Comment:
Nice to see that you added another content part to the view!

Just make sure that the view actually fits nicely on a regular-sized iPhone, on the iPhone 15 Pro it is already quite packed.